aten/src/ATen/native/Convolution.cpp

```diff
@@ -467,8 +467,19 @@ struct ConvParams {
       // always use cudnn_depthwise for channels_last format
       return true;
     }
+    static long cudnn_version = detail::getCUDAHooks().versionCuDNN();
```

Comment:
This static variable is now run even if CUDNN is not enabled or compiled with, is this the source of the ROCM issues? Might not be able to be static as a result (unless you can put it in a constexpr or something).